### PR TITLE
tests: turn off build_cli example doc test

### PR DESCRIPTION
**Pull request by andrewmiller1**
Will let travis stay calm

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -63,7 +63,7 @@ use std::path::PathBuf;
 /// First, it helps if we separate out our `App` definition into a separate file. Whether you
 /// do this as a function, or bare App definition is a matter of personal preference.
 ///
-/// ```
+/// ```ignore
```
Comment (spacekookie) [inline src/lib.rs]: `ignore` actually only doesn't run tests on the CLI. This does not affect doc tests

**Review by spacekookie — DISMISSED** [review]
Hey there, thanks for looking into this :+1: :heart: 

I think if you replace `ignore` with `norun`, that should fix the doc test. `ignore` flags are (AFAIK) only consumed when running normal tests, not doc tests

**Comment by Dylan-DPC-zz** [comment]
@spacekookie i have made the change instead of waiting for them.

**Comment by pksunkara** [comment]
I will fix when moving this repo to main clap workspace.

**Pull request closed without merging**